posthog/queries/test/__snapshots__/test_lifecycle.ambr

```diff
@@ -148,8 +148,8 @@
   '
   WITH 'day' AS selected_period,
        periods AS
-    (SELECT dateSub(day, number, dateTrunc(selected_period, toDateTime('2020-01-19 23:59:59', 'US/Pacific'))) AS start_of_period
-     FROM numbers(dateDiff('day', dateTrunc('day', toDateTime('2020-01-12 00:00:00', 'US/Pacific')), dateTrunc('day', toDateTime('2020-01-19 23:59:59', 'US/Pacific') + INTERVAL 1 day))))
+    (SELECT dateSub(day, number, dateTrunc(selected_period, toDateTime('2020-01-18 23:59:59', 'US/Pacific'))) AS start_of_period
+     FROM numbers(dateDiff('day', dateTrunc('day', toDateTime('2020-01-11 00:00:00', 'US/Pacific')), dateTrunc('day', toDateTime('2020-01-18 23:59:59', 'US/Pacific') + INTERVAL 1 day))))
```

Comment:
This seems like a bug? The date ranges in the lifecycle tests do seem to be 12-19, not 11-18... 🤔

Reply:
12-19 00:00Z is 12-18 in US/Pacific.

I confirmed that the this now just reflects how the date calculations work: Send a query from frontend that has e.g. `date_to: 2023-12-01T06:00:00.000Z` while having US/Pacfic time on the team, and the SQL query in the console will show the day before.